build/main.go

```diff
@@ -92,6 +99,22 @@ type NativeAmount struct {
 	Amount string
 }
 
+type Path struct {
+	Assets []Asset
+}
+
+// PathDestination is a mutator that configures a path_payment's destination asset and amount
+type PathDestination struct {
```

Comment:
As mentioned, this doesn't need to exist.  Someone should simply use the normal `Destination` mutator along with one of `NativeAmount` or `CreditAmount` to replace the functionality of this mutator.